debian/control.aws.in

```diff
@@ -31,12 +31,14 @@ Standards-Version: 4.1.2
 Package: delphix-kernel-@@KVERS@@
 Provides: delphix-kernel-aws, delphix-kernel
 Architecture: any
-Depends: linux-aws,
-         linux-tools-aws,
+Depends: linux-image-@@KVERS@@,
```

Comment:
It looks like with this change, all platform-specific kernel packages share the exact same set of dependencies. Is there a way to consolidate and reduce the duplication?

Reply:
Yeah, we can use a single file instead. I wanted to hold on this for now in case we need to add other packages for some of the platforms, but I can also go ahead with that change if you think it's better to do it now.